src/biogeochem/CNCStateUpdate3Mod.F90

```diff
@@ -177,8 +201,26 @@ subroutine CStateUpdate3( num_soilc, filter_soilc, num_soilp, filter_soilp, &
               cf_veg%m_gresp_xfer_to_fire_patch(p) * dt
          cs_veg%gresp_xfer_patch(p) = cs_veg%gresp_xfer_patch(p) -                 &
               cf_veg%m_gresp_xfer_to_litter_fire_patch(p) * dt
+        else
+         cs_veg%gresp_storage_patch(p) = cs_veg%gresp_storage_patch(p) -           &
+              cf_veg%m_gresp_storage_to_fire_patch(p) * dt
+         cs_veg%gresp_storage_patch(p) = cs_veg%gresp_storage_patch(p) -           &
+              cf_veg%m_gresp_storage_to_litter_fire_patch(p) * dt
+         cs_veg%gresp_xfer_patch(p) = cs_veg%gresp_xfer_patch(p) -                 &
+              cf_veg%m_gresp_xfer_to_fire_patch(p) * dt
+         cs_veg%gresp_xfer_patch(p) = cs_veg%gresp_xfer_patch(p) -                 &
```

Comment:
This line is duplicated for both if and else. Please remove all the duplicated lines. 
For example lines 211-212 are exactly identical to lines 202-203.

Reply:
done